conda_build/build.py

```diff
@@ -96,7 +96,7 @@
     import conda_build.windows as windows
 
 if "bsd" in sys.platform:
-    shell_path = "/bin/sh"
```

Comment:
There is no `/bin/sh` on BSD? Or is this just pointing to the correct location of bash?

Reply:
FreeBSD has /bin/sh but it is not bash.  Since many recipes use bash-isms it is easier to use bash to run the build scripts that adjust the recipe.

Reply:
Makes sense, thanks for elaborating!